doc/Support/FAQ.md

```diff
@@ -368,7 +368,10 @@ adding to /etc/snmp/snmpd.conf :
 
 ## <a name="faq8"> How do I add support for a new OS?</a>
 
-Please see [Supporting a new OS](../Developing/Support-New-OS.md)
+Please see [Supporting a new OS](../Developing/Support-New-OS.md) if you are adding all
+the support yourself, i.e. writing all of the supporting code. If you are only able
```

Comment:
I'd like to see this encourage people to write it themselves.  Most of the time it isn't even code, just some yaml files.  There is a huge backlog of device support because we don't have many users contributing device support for other users.

Reply:
Understood, but we have to be realistic as well. The PR came from a discussion in Discord where a user was confused as to which tutorial to follow. Declared himself a newbie, and asked what he can do to improve. So even though for some it might only be "just some yaml files" for others it feels like a monumental task to accomplish, and they would rather not do anything than do something wrong. Laf suggested he update the docs to improve the wording regarding the differentiation between the 2 methods of adding new OS support.